docs/why.md

```diff
@@ -1,38 +1,155 @@
-## Django's Atomic
-Django's `atomic` ensures database changes are committed together-or-not-at-all. It creates a savepoint or a transaction depending on two factors:
+# Django's Atomic
 
-- The arguments passed to it (`durable` and `savepoint`).
+Django's `atomic` ensures database changes are committed together-or-not-at-all.
+It creates a savepoint or a transaction depending on two factors:
+
+- The arguments passed to it (`durable=` and `savepoint=`).
 - If a database transaction is already open.
 
+## Behaviours
+
 Specifically, the **Behaviours** which `atomic` exhibits are:
```

Comment:
Thought: In context, perhaps the worth "Specifically" is now out of place here.

Reply:
Yes. Could just remove it?